Source/Core/VideoBackends/OGL/OGLMain.cpp

```diff
@@ -74,8 +74,44 @@ std::string VideoBackend::GetDisplayName() const
     return _trans("OpenGL");
 }
 
-void VideoBackend::InitBackendInfo()
+void VideoBackend::InitBackendInfo(const WindowSystemInfo& wsi)
 {
+  {
+    std::unique_ptr<GLContext> temp_gl_context =
+        GLContext::Create(wsi, g_Config.stereo_mode == StereoMode::QuadBuffer, true, false,
+                          Config::Get(Config::GFX_PREFER_GLES));
+
+    FillBackendInfo(temp_gl_context.get());
+  }
+}
```

Comment:
Two issues here:

1. There's no reason for this to be an indented block. I think I must have done that for scoping reasons initially and then moved the code that cared about it away afterwards.
2. This breaks if `temp_gl_context` is `nullptr`, which is the case on my hacky WSL setup. `VideoBackend::Initialize` checks for that.

Reply:
Great point about number 2, sorry I didn't catch that, too used to seeing exceptions and didn't consider `Create()` returning nullptr.

I would have thought _somewhere_ in `GLExtensions::Init` it would check for nullptr..

Reply:
More annoyingly, we don't convey this to the user in any way - `VideoBackend::Initialize` returning false gives ["Failed to initialize video backend!"](https://github.com/dolphin-emu/dolphin/blob/ebbbdc49ab3412693c40add3a56183926e0559b7/Source/Core/Core/Core.cpp#L560-L564) on starting, but there's no equivalent for `VideoBackend::FillBackendInfo` returning false, and it'd be nice to clarify that the context creation failed versus other things (and `GLContext::Create` failing because it couldn't find a matching ifdef versus `context->Initialize` failing).